packages/venia-concept/src/components/Checkout/form.js

```diff
@@ -144,19 +145,23 @@ class Form extends Component {
         return (
             <Fragment>
                 <div className={classes.body}>
-                    <Section label="Ship To" onClick={this.editAddress}>
+                    <Section
+                        label="Ship To"
+                        filledOption={isShippingInformationReady}
```

Comment:
On first glance, it's not clear what `filledOption` means / wants.

Can we think of a more informative name for this prop?